armi/physics/neutronics/latticePhysics/latticePhysicsWriter.py

```diff
@@ -271,7 +280,7 @@ def _getAllNuclidesByCategory(self, component=None):
             nucCategory = ""
             # Remove nuclides from detailed fission product dictionary if they are a part of the core materials
             # (e.g., Zr in the U10Zr which is at fuel temperature and Mo in HT9 which is at structure temp)
-            if nuc in dfpDensities:
+            if nuc in list(dfpDensities):
```

Comment:
del-ing a couple lines before end generally dont want to del as you iterate over keys

Reply:
The iteration is over `for nucName, dens in zip(nuclides, numDensities):`, it's not iterating over `dfpDensities`?

This doesn't hurt but I don't see the need for it.

Reply:
too early in morning. thanks.